### DISCUSSION TITLE
latexindent logs unavailable

### DISCUSSION THREAD

**Discussion by taha-yassine** [James-Yu/LaTeX-Workshop#3596] [Q&A] [answered]
I'm trying to customize the behavior of latexindent. In order to debug the process, I need access to the logs of LaTeX Workshop to see how it launches the script. However, I can't find any line related to latexindent in the logs as you can see on the screen. I know it works because I have `editor.formatOnSave` set to true and the file is correctly formatted when I save it.

![image](https://user-images.githubusercontent.com/40228615/209321372-9916c94b-acf1-46b8-aca1-30ee5a6d7000.png)

**Comment by jlelong** [comment, marked as answer]
Before calling `latexindent`, the extension writes `Start formatting with latexindent` to the logs. If it does not appear, it mostly means that `lateindent` is not called. You can also look up `Format with command` or `Format args` in the logs.

**Reply by taha-yassine** [reply]
You are correct. It ended being the official LaTeX extension that handled the formatting. Thanks for your help.
